src/options.rs

```diff
@@ -48,11 +48,17 @@ pub struct Options {
     pub filter_policy: filter::BoxedFilterPolicy,
 }
 
+#[cfg(feature = "fs")]
+type CurrentEnv = crate::disk_env::PosixDiskEnv;
```

Comment:
can you change it to `DefaultEnv` to clarify that it is not the one and only environment, but merely the default?